task3.py

```diff
@@ -0,0 +1,26 @@
+def is_prime_number(number):
+    if number == 2:
+        return True
+    if not number % 2:
```

Comment:
Не очень явная запись. Более традиционная запись предпочтительна. Python чтит традиции
```suggestion
    if number % 2 == 0:
```